src/test/java/seedu/address/model/person/TutorialGroupContainsKeywordsPredicateTest.java

```diff
@@ -45,30 +45,26 @@ public void equals() {
     public void test_tutorialGroupContainsKeywords_returnsTrue() {
         // One keyword
         TutorialGroupContainsKeywordsPredicate predicate =
-                new TutorialGroupContainsKeywordsPredicate(Collections.singletonList("912345"));
-        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("912345").build()));
+                new TutorialGroupContainsKeywordsPredicate(Collections.singletonList("T19"));
+        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("T19").build()));
 
         // Only one matching keyword
-        predicate = new TutorialGroupContainsKeywordsPredicate(Arrays.asList("9123456", "908976"));
-        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("908976").build()));
-
-        // Non-matching keyword
-        predicate = new TutorialGroupContainsKeywordsPredicate(Arrays.asList("908976", "9123456"));
-        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("9123456908976").build()));
+        predicate = new TutorialGroupContainsKeywordsPredicate(Arrays.asList("T19", "908976"));
+        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("T19").build()));
     }
 
     @Test
     public void test_tutorialGroupDoesNotContainKeywords_returnsFalse() {
         // Zero keywords
         TutorialGroupContainsKeywordsPredicate predicate =
                 new TutorialGroupContainsKeywordsPredicate(Collections.emptyList());
-        assertFalse(predicate.test(new PersonBuilder().withTutorialGroup("908976").build()));
+        assertFalse(predicate.test(new PersonBuilder().withTutorialGroup("T19").build()));
 
-        // Keywords does not match name
+        // Keywords does not match tutorial group
         predicate = new TutorialGroupContainsKeywordsPredicate(
                 Arrays.asList("99999", "123452354", "12345235", "12345253"));
```

Comment:
Perhaps we can follow the "[A-Z]\d{2}" format here.
predicate = new TutorialGroupContainsKeywordsPredicate(
                Arrays.asList("T20", "T21", "T22", "T23"));